pyspecdata/lmfitdata.py

```diff
@@ -180,27 +180,20 @@ def _taxis(self, taxis):
             taxis = np.linspace(taxis[0], taxis[1], 300)
         return taxis
 
-    def eval(self, taxis=None, set_what=None, set_to=None):
+    def eval(self, taxis=None):
         """Calculate the fit function along the axis taxis.
 
         Parameters
         ----------
         taxis: ndarray, int
             :if ndarray: the new axis coordinates along which we want to calculate the fit.
             :if int: number of evenly spaced points along the t-axis along the fit
-        set_what: 'str', optional
-            forcibly sets a specific symbol
-        set_to: double, optional
-            the specific value(int) you are assigning the symbol you included
 
         Returns
         -------
         self: nddata
             the fit function evaluated along the axis coordinates that were passed
         """
-        if isinstance(set_what, dict):
-            set_to = list(set_what.values())
-            set_what = list(set_what.keys())
```

Comment:
here, I'm just removing obsolete stuff (lmfit allows us to fix parameters by setting the `vary` property of a parameter to `False`)

Reply:
sounds good